src/xcode/ENA/ENA/Source/Views/StackViewLabel.swift

```diff
@@ -0,0 +1,89 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+
+class StackViewLabel: UIView {
```

Comment:
There is no stack view involved inside of StackViewLabel.
Rename suggestion.

```suggestion
class ViewWrappedLabel: UIView {
```